stellar/deploy-contract.js

```diff
@@ -8,12 +8,77 @@ const { getDomainSeparator, getChainConfig } = require('../common');
 const { prompt, validateParameters } = require('../common/utils');
 const { weightedSignersToScVal } = require('./type-utils');
 const { ethers } = require('hardhat');
-const { readFileSync } = require('fs');
+const { writeFileSync, readFileSync } = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
 const {
     utils: { arrayify, id },
 } = ethers;
 require('./cli-utils');
 
+const AXELAR_RELEASE_BASE_URL = 'https://static.axelar.network/releases/axelar-cgp-stellar';
+
+const SUPPORTED_CONTRACTS = new Set([
+    'axelar_gateway',
+    'axelar_operators',
+    'axelar_gas_service',
+    'interchain_token',
+    'token_manager',
+    'interchain_token_service',
+    'upgrader',
+]);
+
+function getWasmUrl(contractName, version) {
```

Comment:
This should be moved to `common/utils` since it'll be relevant for other contracts such as cosmwasm as well